backend/backend/src/main/java/de/aaaaaaah/velcom/backend/access/BenchmarkReadAccess.java

```diff
@@ -51,8 +54,20 @@
  */
 public class BenchmarkReadAccess {
 
-	protected static final Caffeine<Object, Object> RUN_CACHE_BUILDER = Caffeine.newBuilder()
-		.maximumSize(10000);
+	protected static Cache<CommitHash, Run> buildRunCache(RepoId repoId) {
```

Comment:
Could we place that method below the fields and probably constructor? I think that's the common structure in every other file.